.ci/pitest.sh

```diff
@@ -47,9 +47,7 @@ pitest-annotation|pitest-design \
 
 pitest-regexp)
```

Comment:
please append this label to previous "case".
We do not need this case block any more.

Reply:
updated